examples/pom.xml

```diff
@@ -35,6 +35,10 @@
     <sbt.project.name>examples</sbt.project.name>
     <build.testJarPhase>none</build.testJarPhase>
     <build.copyDependenciesPhase>package</build.copyDependenciesPhase>
+    <flume.deps.scope>provided</flume.deps.scope>
```

Comment:
Just double-checking, these are still needed in order for the examples assembly to set the right scope for other components when it builds, since these values are used and can vary elsewhere? LGTM